tests.py

```diff
@@ -9,6 +9,8 @@
 
 
 class ImportsTest(unittest.TestCase):
+    maxDiff = None
```

Comment:
TIL.   keep this as None?

Reply:
I think so, otherwise if we get an error the output is 
>stuff is different, the diff is X long that's more than maxDiff, set maxDiff to None to view it

so we may as well keep it none